### PR TITLE
[76] Set OPENSSL_API_COMPAT=10100

### PR DESCRIPTION
**Pull request by JustinKyleJames**
This is to fix the warning for deprecated methods.

### CODE REVIEW

**Review by alanking — APPROVED** [review]
I think this is fine for the release. Probably will get cleaned up further in #74

**Review by korydraughn — APPROVED** [review]
Looks good to me.

Let's wait for one more approval before pounding.

**Comment by trel** [comment]
ah, nice.

File: CMakeLists.txt [hunk 1]
```diff
@@ -187,7 +187,12 @@ set(CPACK_RPM_RELOCATION_PATHS $ENV{DEST_LIB_DIR})
...
-set(CPACK_DEBIAN_PACKAGE_DEPENDS "${IRODS_PACKAGE_DEPENDENCIES_STRING}, irods-runtime (= ${IRODS_VERSION}), libc6, libssl1.1")
+
+if (OPENSSL_VERSION VERSION_LESS "3.0.0")
+    set(CPACK_DEBIAN_PACKAGE_DEPENDS "${IRODS_PACKAGE_DEPENDENCIES_STRING}, irods-runtime (= ${IRODS_VERSION}), libc6, libssl1.1")
+else()
+    set(CPACK_DEBIAN_PACKAGE_DEPENDS "${IRODS_PACKAGE_DEPENDENCIES_STRING}, irods-runtime (= ${IRODS_VERSION}), libc6, libssl3")
+endif()
 
 set(CPACK_RPM_PACKAGE_NAME ${IRODS_CLIENT_NAME})
 if (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "centos" OR IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "centos linux")
```
Comment (alanking) [inline CMakeLists.txt:198]: Consider removing this check as well. We "assume" Enterprise Linux for RPM builds in our other repos now: https://github.com/irods/irods/commit/8dd1b287ce1a24474032787ca7996b2a2baab330#diff-5b5b3e80ba4ffcdf6c8af4e0b8b6d640d9ec0284903e2d83846321e7d2a88a02R94-R97

**Review by SwooshyCueb — APPROVED** [review]

**Comment by JustinKyleJames** [comment]
Removed the draft.  The latest pushes build on all of the environments that previously had problems.

File: CMakeLists.txt (hunk 1 above)
Comment (alanking) [reply CMakeLists.txt:201]: What I mean is that the logic captured by this if statement should be the default (i.e. make it the `else` case).

If this is working on the supported platforms for Globus servers then maybe we don't worry about it. I defer.

**Comment by JustinKyleJames** [comment]
Pushed another commit based on Alan's suggestion.  The build has been tested with the latest CMakeLists.txt on all environments.

**Review by alanking — APPROVED** [review]
Yep. Let's get one more approval and then squash and #

**Review by korydraughn — APPROVED** [review]
Looks good.

Squash and pound it.

**Comment by JustinKyleJames** [comment]
> Looks good.
> 
> Squash and pound it.

Done

**Pull request merged**